README.md

```diff
@@ -173,3 +173,23 @@ con = td.connect(apikey=apikey, endpoint=endpoint, writer=writer)
 df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 10]})
 td.to_td(df, 'mydb.buzz', con, if_exists='replace', index=False)
 ```
+
+## For developers
+
+We use [black](https://black.readthedocs.io/en/stable/) and [isort](https://github.com/timothycrosley/isort) as a formatter, and [flake8](http://flake8.pycqa.org/en/latest/) as a linter. Our CI checks format with them.
+
+We highly recommend you to introduce [pre-commit](https://pre-commit.com/) to ensure your commit follows required format.
+
+You can install pre-commit as follows:
```

Comment:
Personally I would also like to mention how to skip pre-commit check here: `git commit --no-verify`

[Recent updates on pandas contributing guide](https://github.com/pandas-dev/pandas/pull/27233) could be a good reference.

Reply:
thanks for the reference. I will add to skip pre-commit check.